src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final ColorSupplier colorSupplier = new ColorSupplier(Color.values());
```

Comment:
```suggestion
    private final ColorSupplier colorSupplier = new ColorSupplier();
```

Reply:
![Zrzut ekranu 2024-05-18 130427](https://github.com/mate-academy/jv-lottery/assets/169672147/88f73a5e-3285-4478-822a-ff8651dba580)
 i have error when i do like this

Reply:
You have an error because you first you have to implement all changes that I suggested.